source/import-font-and-images-assets/index.html.md.erb

```diff
@@ -12,28 +12,30 @@ To use the font and image assets from GOV.UK Frontend, you can either:
 
 ## Serve the assets from the GOV.UK Frontend assets folder - recommended
 
-Set up your routing so requests for files in `<YOUR-SITE-URL>/assets` are served from `/node_modules/govuk-frontend/dist/govuk/assets`.
+<%= warning_text('The file paths on this page point to asset folders that were updated in GOV.UK Frontend v5.10.0') %>
+
+Set up your routing so requests for files in `<YOUR-SITE-URL>/assets/rebrand` are served from `/node_modules/govuk-frontend/dist/govuk/assets/rebrand/rebrand`.
```

Comment:
```suggestion
Set up your routing so requests for files in `<YOUR-SITE-URL>/assets/rebrand` are served from `/node_modules/govuk-frontend/dist/govuk/assets/rebrand`.
```

I may need a sense check on this from a dev - my read is that the asset path on the GOV.UK template is set to `/assets/rebrand` if the rebrand is on, so we want folks to set up that folder rather than use `<YOUR-SITE-URL>/assets`